app/models/course_data/courses_users.rb

```diff
@@ -101,6 +101,7 @@ def update_cache
     self.character_sum_ms = character_sum(revisions, Article::Namespaces::MAINSPACE)
     self.character_sum_us = character_sum(revisions, Article::Namespaces::USER)
     self.character_sum_draft = character_sum(revisions, Article::Namespaces::DRAFT)
+    self.refs_sum_ms = references_sum(revisions)
```

Comment:
You should run `annotate` so that the comments at the top of the file get updated with the new attribute as well.